src/EnergyPlus/ScheduleManager.cc

```diff
@@ -562,7 +562,7 @@ namespace ScheduleManager {
                                 CSVAllColumnNameAndValues[colCnt - 1] = timestepColumnValues;
                             }
                         } else {
-                            columnValue = std::stod(subString, nullptr);
```

Comment:
It is worth noting that this function is faster on my local machine while having no diffs. 
Ran in 5.74 seconds. 
![image](https://user-images.githubusercontent.com/45446967/121533321-6840a100-c9bd-11eb-8523-001749ccc105.png)

Reply:
But, we would need to handle errors here, and I can explore that if this is something that is interests us.

Reply:
Which file were you using here?

Reply:
Oops, forgot to mention that. The `SolarShadingTest_ImportedShading.idf` file.

Reply:
Umm, what is up with my computer??? 2 min 28 sec...

Reply:
And with using with the ```std::stod``` option, about 15 sec... what in the world is going on?

Reply:
Probably one is develop and the other one is release config. And probably you have a much better computer than I do 😄

Reply:
I tested both in release configuration.

Reply:
Nope, I double-checked, rebuilt, and ran multiple times. I'm in release mode on Mac. With ```UtilityRoutines::ProcessNumber``` this file is taking about 2:30-3:00 min; with ```std::stod```, it's about 15 sec.

Reply:
That's very interesting... For me, Linux release build on CLion, `UtilityRoutines::ProcessNumber` takes 8 seconds vs `std::stod` takes 5ish.